Tests/_CryptoExtrasTests/TestRSABlindSigning.swift

```diff
@@ -202,4 +202,26 @@ final class TestRSABlindSigning: XCTestCase {
             parameters: .RSABSSA_SHA384_PSS_Randomized
         )
     }
+
+    func testConstructAndUseKeyFromRSANumbersWhileRecoveringPrimes() throws {
+        let data = Array("hello, world!".utf8)
+
+        // Generating primes is probabilistic, so we run this test multiple times.
```

Comment:
I'm a bit nervous about a test that _fails_ probabilistically under correct scenarios, which is what this test appears to do. We should presumably make an estimate of what the failure rate should be, scaled by the number of test iterations here, and then move the threshold of failures to two std dev.

Reply:
Actually I added this because with the first commit of the algorithm it was failing something like once every 5 tries. This was however due to me adding a wrong exponentiation which is now resolved. I've run this test locally a handful of times (even with 1000 iterations) and it always succeeded, hence I'm actually quite confident the algorithm works fine and we can remove both the comment and the loop

Reply:
Let's remove the loop.

Reply:
Done!